### PR TITLE
Fix maintainers

### PR DESCRIPTION
**Pull request by isuruf**
cc @jakirkham, this was the reason for https://travis-ci.org/conda-forge/conda-forge.github.io/jobs/276681990. we should have better error handling

### CODE REVIEW

**Comment by jakirkham** [comment]
Not totally overlapping, but close to issue ( https://github.com/conda-forge/conda-forge-webservices/issues/80 ). This has happened a couple of times unfortunately.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -49,5 +49,5 @@ about:
 
 extra:
   recipe-maintainers:
-    - Eric89GXL
+    - larsoner
```
Comment (jakirkham) [inline recipe/meta.yaml]: Please confirm this is correct, @larsoner.

Comment (isuruf) [reply recipe/meta.yaml]: I added this because of https://github.com/conda-forge/staged-recipes/pull/2396#discussion_r100676519

**Comment by larsoner** [comment]
Yes, that's my changed name. Sorry for the inconvenience!

**Comment by jakirkham** [comment]
np. Thanks for letting us know.

Can I ask that you please update any other feedstocks that you are a maintainer on, @larsoner? Thanks.

**Comment by larsoner** [comment]
I was also on PySurfer so I changed that one, thanks

**Comment by isuruf** [comment]
Can somebody please merge this?

**Comment by jakirkham** [comment]
The AppVeyor failure can be fixed by [re-rendering]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ) with `conda-smithy` 2.4.0.

**Comment by isuruf** [comment]
Thanks

**Comment by jakirkham** [comment]
np. Thanks for catching and addressing the issue.

**Pull request merged**